src/main/java/randoop/sequence/VariableRenamer.java

```diff
@@ -23,31 +26,40 @@ public VariableRenamer(Sequence sequence) {
    * examples:
    *
    * <pre>
-   *   int var0 = 1     becomes  int i0 = 1
-   *   ClassName var0 = new ClassName()      becomes ClassName className = new ClassName()
-   *   Class var0 = null      becomes Class clazz = null
+   *   int var0 = 1 becomes  int int0 = 1
+   *   ClassName var0 = new ClassName() becomes ClassName className = new ClassName()
+   *   Class var0 = null becomes Class cls = null
+   *   Queue<Set<List<Comparable<String>>>> var0 = null becomes Queue<Set<List<Comparable<String>>>> listSetQueue = null
+   *   ArrayList<String> var0 = null becomes ArrayList<String> strList = null
    * </pre>
    *
    * @param type the type to use as base of variable name
    * @return a variable name based on its type, without a trailing number
    */
   static String getVariableName(Type type) {
+    return getVariableName(type, 0);
+  }
 
+  /**
+   * Heuristically renames each variable to a name that is based on the variable's type.
+   *
+   * @param type the type to create a variable name for
+   * @param depth the number of components (i.e. type arguments) of the type that have been used to
```

Comment:
A more natural convention for the parameter `depth` would be the depth to which to explore, not the depth so far.  That is, call the private method with `VAR_NAME_MAX_DEPTH`, and make the private method test against 0 and decrement rather than increment `depth` on recursive calls.
That makes the private method understandable on its own, without reference to `VAR_NAME_MAX_DEPTH`.